server/docs/hellobooks_api_doc.yaml

```diff
@@ -588,6 +589,33 @@ paths:
               schema:
                 $ref: "#/components/schemas/errorResponse"
 
+  /books/extend/{id}:
+    get:
```

Comment:
Why are you using a `GET` request in the documentation? This would throw an error of `endpoint not found`

Reply:
Sorry. I would change it now. Thanks